src/modules/standard-guidelines/auth-module/AuthController.ts

```diff
@@ -0,0 +1,38 @@
+import { Router, Request } from "express";
+import proxy = require("express-http-proxy");
+import { Controller } from "../../../interfaces/Controller";
+
+const STANDARD_GUIDELINES_API = process.env.STANDARD_GUIDELINES_API || 'localhost:8888';
+
+export class AuthController implements Controller {
+    buildRouter(): Router {
+        const router = Router();
+        
+        /**
+         * @swagger
+         * /authenticate:
+         *  get:
+         *    description: authenticates a user 
+         *    tags:
+         *      - Standard Guideline Service
+         *    responses:
+         *      200:
+         *        description: OK
+         *      401:
+         *        description: INVALID ACCESS - User does not have a valid certificate
+         *      403:
+         *        description: FORBIDDEN - User is not whitelisted, please use a whitelisted certificate
```

Comment:
It doesn't throw these at all with this route, only with the routes that require authentication.  It returns an optional token (if they have the cert) and redirects them to the standard guidelines client regardless if they are authenticated or not.